### DISCUSSION TITLE
Drag-and-drop not working when password input has focus

### DISCUSSION THREAD

**Discussion by ericshulman** [TiddlyWiki/TiddlyWiki5#5972] [General]
I'm not sure if this is a bug, but it seems odd to me...

(Tested on TiddlyWiki.com)
1. From the sidebar, grab any tiddler title (e.g., "HelloThere") and begin dragging.  Note the cursor change that indicates a "drag-in-progress".  This shows that drag-and-drop is working normally.  Drop the link anywhere in the background, or press <kbd>escape</kbd> to cancel the drag.
2. Create a tiddler containing: `<$edit-text tiddler="TryIt" tag="input" type="password" />`
3. Put focus into rendered input control and repeat test from step 1.  Note that **there is no cursor change, and drag-and-drop seems to be broken.**
4. Click anywhere outside of input control to remove focus
5. Repeat test from step 1.  Note cursor change now occurs, showing that drag-and-drop is again working normally.
6. Edit tiddler from step 2 and remove type="password" argument from widget (i.e., make it a normal text input control)
7. Put focus into rendered input control and repeat test from step 1.  Note cursor change occurs, showing that drag-and-drop is working normally.

Thus, it seems that two factors result in drag-and-drop failure:
A) Input control has type="password", and
B) Input control has focus when drag starts.

Also, using browser developer console, I inspected the "HelloThere" link immediately after the failed drag in step 3, and it showed "tc-dragging" class was added, indicating that drag operation did start, but then failed somewhere along the way.

**Comment by Jermolene** [comment]
Hi @ericshulman I can confirm the same behaviour in Chrome, but not Firefox and Safari, so I guess it's a Chrome bug?
